docs/appendix/glossary/rdk.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: RDK (Robot Development Kit)
 id: rdk
-full_link:
-short_description: The official Viam-developed codebase that provides all functionality of an SDK and more.
+full_link: /program/rdk/
+short_description: The open-source, on-robot portion of the Viam platform, that provides viam-server and the Go SDK.
```

Comment:
```suggestion
short_description: The open-source, on-robot portion of the Viam platform, that provides `viam-server` and the Go SDK.
```

Reply:
Formatting in these doesn't do anything I'm afraid